graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/CodegenTestClassLoader.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ *
+ *  Copyright 2020 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.graphql.dgs.codegen
+
+import com.google.testing.compile.Compilation
+import java.io.ByteArrayOutputStream
+import java.util.*
+import javax.tools.JavaFileObject
+
+internal class CodegenTestClassLoader(private val compilation: Compilation, parent: ClassLoader?) : ClassLoader(parent) {
```

Comment:
This _test_ Classloader can be used when interacting with any of the classes available in the given compilation. Even if the class depend on others defined in codegen. This is not the case with the `compileAndGetClass` method in `ClientApiGenTest`.

Reply:
Very cool, you got the classloader test set up!